### PR TITLE
Adds DateTimePicker to Add_Course

### PR DESCRIPTION
**Pull request by Akshen**
Adds DatetimePicker to start_enroll_time and end_enroll_time field
Fixes Add Course and Cancel Button's CSS

### CODE REVIEW

File: yaksh/templates/yaksh/add_course.html [hunk 1]
```diff
@@ -13,9 +19,14 @@
...
+  <br/>
+  <script type="text/javascript">
+    $("#id_start_enroll_time").datetimepicker({format: 'Y-m-d H:i:s'});
+    $("#id_end_enroll_time").datetimepicker({format: 'Y-m-d H:i:s'});
+  </script>
   </center>
 
-  <center><button class="btn primary" type="submit" id="submit" name="questionpaper">Add Course</button>
-  <button class="btn primary" type="button" name="button" onClick='location.replace("{{URL_ROOT}}/exam/manage/");'>Cancel</button> </center>
+  <center><button class="btn btn-default" type="submit" id="submit" name="questionpaper">Add Course</button>
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/add_course.html]: Let the button name be **Save** and not **Add Course**.

File: CHANGELOG.txt [hunk 2]
```diff
@@ -12,6 +12,7 @@
 * Fixed a bug that did not allow expected input in Standard I/O type question to be none.
 * UI changes in grade user, view answerpaper and monitor pages.
 * Fixed a bug that would require shebang to be put in for Bash assertion based questions.
+* Add Datetime Picker to Add Course Page
```
Comment (maheshgudi) [inline CHANGELOG.txt]: For continuity's sake, let the sentence be in past tense.

File: yaksh/templates/yaksh/add_course.html [hunk 3]
```diff
@@ -4,6 +4,12 @@
 
 {% block css %}
 <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/question_quiz.css" type="text/css" />
+<link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/jquery.datetimepicker.css" type="text/css" /> 
+{% endblock %}
+
+{% block script %}
+<script src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.9.1.min.js"></script>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/add_course.html]: This is already included in the base html.

**Comment by prathamesh920** [comment]
LGTM

**Pull request merged**